### PR TITLE
Fix broken external links

### PR DESCRIPTION
**Pull request by Josh-Cena**
This PR fixes all external links that are 403, 404, or generate a TLS error, by either downgrading to HTTP, removing the link, or replacing with a working version.

### CODE REVIEW

File: files/en-us/learn/forms/how_to_build_custom_form_controls/index.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ Designing new interactions is generally only an option for very large industry p
 It is best not to invent new user interactions. For any interaction you do add, it is vital to spend time in the design stage; if you define a behavior poorly, or forget to define one, it will be very hard to redefine it once the users have gotten used to it. If you have doubts, ask for the opinions of others, and if you have the budget for it, do not hesitate to [perform user tests](https://en.wikipedia.org/wiki/Usability_testing). This process is called UX Design. If you want to learn more about this topic, you should check out the following helpful resources:
 
 - [UXMatters.com](https://www.uxmatters.com/)
-- [UXDesign.com](https://uxdesign.com/)
+- [UXDesign.com](http://uxdesign.us/)
```
Comment (wbamberg) [inline files/en-us/learn/forms/how_to_build_custom_form_controls/index.md:79]: Unless they are really essential, can we just remove HTTP-only links?

File: files/en-us/games/techniques/control_mechanisms/other/index.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ Leapmotion is becoming more and more popular due to very good integration with V
...
+You can check out the tutorial about [using the Leap Motion plugin for Kiwi.js](https://webdesign.tutsplus.com/add-motion-control-to-a-kiwijs-game-with-the-leap-motion-controller--cms-20455t), or the case study of [building a web game with Leap Motion and Pixi.js](https://arstechnica.com/information-technology/2014/04/building-a-gesture-controlled-web-game-with-leap-motion/). Be sure to visit the [LeapJS repository on GitHub](https://github.com/leapmotion/leapjs) to learn about the JavaScript client for the Leap Motion controller and read the documentation there. If all else fails, there's also a [gallery of working examples](https://leap2.ultraleap.com/downloads/) you can look at.
```
Comment (wbamberg) [inline files/en-us/games/techniques/control_mechanisms/other/index.md:79]: This Games content is so horribly out of date. Although the Kiwi.js tutorial is still a real link, the Kiwi.js site itself now redirects to something scummy-looking, and it doesn't seem like any of the LeapMotion content here is going to be usable any more. Still, technically out of scope for this PR I suppose.

```suggestion
```

**Comment by Josh-Cena** [comment]
I really hope there's a way to automatically discover hijacked sites 😅

**Review by wbamberg — APPROVED** [review]
👍 thank you @Josh-Cena !

**Pull request merged**